tests/integration/targets/openssl_pkcs12/tasks/impl.yml

```diff
@@ -4,6 +4,51 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 - block:
+  - name: "({{ select_crypto_backend }}) Generate PKCS#12 file from contents"
```

Comment:
I just added this minimal test because the other tests basically test the new behavior as well (since everything is basically redirect to `content` now). What worries me a bit about the tests is that I don't understand how for instance 
```
name: "({{ select_crypto_backend }}) Generate PKCS#12 file with multiple certs and passphrase, again (idempotency)"
```
is actually checking idempotency. It does register `p12_multiple_certs_idempotency` but never checks that the result is unchanged (cannot find that the variable is used at all)?

Reply:
Did you check `tests/integration/targets/openssl_pkcs12/tests/validate.yml`?

Reply:
Outch, that is embarrassing, I saw some asserts in `impl.yml` like:
```
  - name: "({{ select_crypto_backend }}) Validate PKCS#12"
    assert:
      that:
      - p12_standard.pkcs12 == ansible_p12_content.content
      - p12_standard_idempotency.pkcs12 == p12_standard.pkcs12
```

and assumed the tests where inline. Sorry for that!

Reply:
Using `tests/validate.yml` for assertions is something that community.crypto (resp. its tests) have been doing for a loong time. I can't say I like it too much, but also not that little that I'd change all of them ;-) Some newer modules simply have all tests in `tasks/main.yml` (or some files in `tasks/`)...